content/learn/quick-start/next-steps.md

```diff
@@ -1,11 +1,9 @@
 +++
 title = "Next Steps"
-weight = 12
-sort_by = "weight"
-template = "docs-section.html"
-page_template = "docs-section.html"
 insert_anchor_links = "right"
 aliases = ["learn/book/next-steps"]
+[extra]
+weight = 3
```

Comment:
This weight was changed from 12 to 3. Was that purposeful?

Reply:
Yes and no. It was originally 4, then made 3 on main during some re-ordering with plugin guidelines, the fact that it is 12 on main means the original new book to main merge fucked up some weights and wasn't caught. It's still supposed to be 3.

Hmm. Either time to open up an issue and fix that on main, or just wait until this is merged to avoid having to solve _another_ merge conflict.

Reply:
I don't think this is a blocking issue, so it can probably be merged as-is.